cpg_workflows/status.py

```diff
@@ -35,12 +35,10 @@ def complete_analysis_job(
         update_analysis_meta (Callable | None): function to update analysis meta
         tolerate_missing (bool): if True, allow missing output
     """
-    import traceback
 
     from cpg_utils import to_path
-    from metamist.apis import AnalysisApi
-    from metamist.exceptions import ApiException
-    from metamist.models import Analysis, AnalysisStatus
+
+    from .metamist import AnalysisStatus, get_metamist
```

Comment:
```suggestion
    from cpg_workflows.metamist import AnalysisStatus, get_metamist
```

I get that relative imports are fine (now? though previously were recommended against), but as it's part of the package is there a reason not to be explicit?

Reply:
Good point, updated.